omnichannel-server/src/feature/message-source/MessageParser.ts

```diff
@@ -0,0 +1,62 @@
+import { MessageBirdPayload } from "./interfaces/MessageBirdPayload";
+import { MessageTwilioPayload } from "./interfaces/MessageTwilioPayload";
+import { MessagePayload } from "./interfaces/MessagePayload";
+
+export const MessageParser = ({body}) => {
+    const messageBody = <MessageBirdPayload|MessageTwilioPayload>body;
+
+    if(isBird(<MessageBirdPayload>messageBody)) {
```

Comment:
Este syntax es válido. 

También se puede usar: 

```suggestion
    if(isBird(messageBody as MessageBirdPayload)) {
```

Reply:
A gracias por la sugerencia